meetings/2024-01-09.md

```diff
@@ -40,13 +44,70 @@
+* SC: Two things: whether this dicussion should live on solid specification repo, we can add more notes.. maybe we should take this procedure and turn it into something like a template/general guideline and put it in the w3c cg solid repo (URL). Copy of this specific procedure under elections/2023, input/output, etc. The other thing is making something reusable. Simplest would be copy/paste this into an .md with links to annoucements. 
+
+ACTION: Move discussion on 2023 election procedure into dedicated document in w3c-cg/solid. SC to PR.
+
+ACTION: Create issue to template the procedure. SC.
+
+* eP: We also have hackmd. Is that just scratchpad now? Anything else we should capture from there. https://hackmd.io/a6jv3bfnQBGace7Ajd_ePw
+* SC: Tie-breaking is something we can review.
+* VB: Have been reading about MeekSTV, not sure if that's most suitable.
+* SC: IIRC, it is used for TAG/AB elections.
+* eP: Perhaps add something about python2? the command-line to explictly use python2 for anyone that's not too familiar. 
+
+
+### Solid Ecosystem Monitor
+URL: https://virginiabalseiro.github.io/solid-ecosystem-monitor/
 
+* VB: Added table with election results data.
+* VB: This monitor will be reorganised. Have pages for meetings, elections, and other stuff.
+* VB: Table on elections results is at the bottom. Counting the votes each candidate received per rank, and percentage of total votes for each rank. We can draw interesting conclusions. Could be useful. Also to evaluate the system re STV, tied to some of htese observations.
+* SC: We said we would move this to solid contrib. 
+* VB: Yes. We can move it. 
+* eP: I objected to having it in the same repo but if you think it's more convenient to have it there.. I'd prefer a dedicated repo. 
+* VB: I think that's fine.
+* SC: No strong preference. 
+* SC: I don't know how to evaluate the results ina meaningful and fair way without undertsnading MeekSTV well. I'll trust the consensus in W3C as to why they use it and keep it aligned to rest of election stuff there. Easier to work with a process that they have than doing out own things, in case we have questions, etc. 
+* SC: That doesn't mean we cannot do additional things on top of things we observe, charter has some constraints on only one person from each affiliation, we can add other contraints on other things. I don't know if there shoulld be some thresholds. Comes down to the math. Looking at these results: one of the winners has 39% unranked. I don't know how to interpret but at face value from 41 ballots 39% said they did not want the candidate but they got a seat. I don't know what to do with that information but it does stand out. It had to do with the method. 
```

Comment:
```suggestion
* SC: I don't know how to evaluate the results in a meaningful and fair way without understanding MeekSTV well. I'll trust the consensus in W3C as to why they use it and keep it aligned to rest of election stuff there. Easier to work with a process that they have than doing our own things, in case we have questions, etc. 
* SC: That doesn't mean we cannot do additional things on top of things we observe. Charter has some constraints on only one person from each affiliation; we can add other constraints on other things. I don't know if there should be some thresholds. Comes down to the math. Looking at these results: one of the winners has 39% unranked. I don't know how to interpret that, but at face value, from 41 ballots, 39% said they did not want the candidate, but they got a seat. I don't know what to do with that information, but it does stand out. It had to do with the method. 
```